src/components/SingleArticle.jsx

```diff
@@ -0,0 +1,35 @@
+import { useParams } from "react-router-dom";
+import { useState, useEffect } from "react";
+import { getSingleArticle } from "../api";
+import { Link } from "react-router-dom";
+import VoteButton from "./VoteButton";
+
+const SingleArticle = () => {
+  const { article_id } = useParams();
+  const [singleArticle, setSingleArticle] = useState([]);
+  useEffect(() => {
+    getSingleArticle(article_id).then((articleFromApi) => {
+      setSingleArticle(articleFromApi);
+    });
+  }, [setSingleArticle, article_id]);
+  return (
+    <main>
+      <h1>
+        <Link to="/">Home page</Link>
+      </h1>
+
+      <h2>{singleArticle.title}</h2>
+      <p>topic: {singleArticle.topic}</p>
+      <p>votes: {singleArticle.votes}</p>
```

Comment:
You have votes showing more than once, in this component and the VoteButton component! Have a think about whether we need both. You are passing in singleArticle.votes which is not a value stored in the state so it will never change.